webapp/src/main/java/com/plhal/ares/webapp/AresApiApplication.java

```diff
@@ -1,15 +1,30 @@
 package com.plhal.ares.webapp;
 
+import com.plhal.ares.model.DataRepository;
+import com.plhal.ares.model.DataRepositoryImpl;
+import com.plhal.ares.service.DataService;
+import com.plhal.ares.service.DataServiceImpl;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
-
-@SpringBootApplication(scanBasePackages = {"com.plhal.ares.webapp", "com.plhal.ares.service", "com.plhal.ares.model"})
+@Configuration
```

Comment:
Kdyz se podivate, tak zjistite, ze `SpringBootApplication` obsahuje i `SpringBootConfiguration` a to obsahuej i `Configuration`. Ta anotace je tu tedy zbytecna.
Nicmene prehlednejsi varianta je vytvorit `AresApiConfig`, tu anotovat jako `Configuration` a zahrnout ji do teto `SpringBootApplication`. Coz vam umozni oddelit tridu pro konfiguraci od spoustece aplikace, coz je pri nejakem cleneni rozrustajici aplikace uzitecne.